### PR TITLE
Autoscroll

### PR DESCRIPTION
**Pull request by chris-yooo**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+MONGODB_URI=mongodb://admin:iulilguzfg867foliO98f@195.201.170.57:20000/chrat?authSource=admin
```
Comment (stefankorder) [inline .env:1]: The .env file should be in your gitignore file and not pushed to github.

File: .gitignore [hunk 2]
```diff
@@ -21,7 +21,7 @@ yarn-debug.log*
 yarn-error.log*
 
 # local env files
-.env
+# .env
```
Comment (stefankorder) [inline .gitignore:24]: The .env file should be in your gitignore file and not pushed to github.

**Review by stefankorder — CHANGES_REQUESTED** [review]
Good work! One important hint.

File: .env (hunk 1 above)
Comment (chris-yooo) [reply .env:1]: jeah but i cant use env variable at vercel i want to try the merge and function after merge at vercel

**Pull request merged**